app/(dashboard)/conversations/page.tsx

```diff
@@ -15,18 +13,7 @@ const Conversations = async () => {
   return (
     userId && (
       <ConversationProvider userId={userId}>
-        <div className='mt-4 flex justify-between px-3 '>
-          <button>
-            <PlusIcon width={45} height={45} />
-          </button>
-          <button>
-            <MeatballIcon width={35} height={35} />
-          </button>
-        </div>
-        <div className='mt-4'>
-          <ConversationsList />
-          <CurrentConversation />
-        </div>
+        <ConversationsWrapper></ConversationsWrapper>
```

Comment:
I really wanted to fight us encapsulating our components further but this seems to make sense 😅 

Can we just make it pretty by making it self-closing ? `<ConversationsWrapper />`

Reply:
großartig ❤️